### DISCUSSION TITLE
Show actions you can already do with current materials

### DISCUSSION THREAD

**Discussion by machasins** [xanthics/genshin_grind_planner#3] [Ideas]
Sometimes I don't realize when looking at the website that I could ascend a character or weapon or level them up to the next barrier. I think having a section somewhere on the character's screen would definitely be nice. Just a place that tells the user that they could ascend Xiangling to 60A, raise Sucrose's skill talent to level 6, or level up their Wolf's Gravestone from 80A to 90. 

Thanks!

**Comment by xanthics** [comment]
I'll take a look at this, see how it affects page responsiveness.  If this is implemented, I'm reluctant to have it consider actions that are possible from converting up mats.
